src/main/java/team/creative/creativecore/common/gui/GuiParent.java

```diff
@@ -42,17 +39,21 @@ public class GuiParent extends GuiControl implements IGuiParent, Iterable<GuiChi
     
     public GuiParent(String name, GuiFlow flow) {
         super(name);
-        this.flow = flow;
+        this.setFlow(flow);
```

Comment:
I don't see a reason to use the setFlow instead of the direct call.

Reply:
reverted